pkg/stores/user/db.go

```diff
@@ -886,3 +886,30 @@ func (s *DBStore) Count(ctx context.Context) (int64, error) {
 func (s *DBStore) CachedCount(ctx context.Context) int {
 	return 0
 }
+
+func (s *DBStore) GetModList(ctx context.Context) (*pb.GetModListResponse, error) {
+	var users []User
+	if result := s.db.
+		Where("is_admin = ?", true).Or("is_mod = ?", true).
```

Comment:
I think we should add an index on these fields. I can do it manually on prod but we can change these lines up top to chain in some admin/mod indices? 

```go
	db.Model(&User{}).
		AddUniqueIndex("username_idx", "lower(username)").
		AddUniqueIndex("email_idx", "lower(email)").
		AddIndex("api_key_idx", "api_key")
```

Reply:
added some indexes, but they don't seem to be used. feel free to revert that commit.

[partial indexes](https://www.postgresql.org/docs/current/indexes-partial.html) may be more appropriate. but, not sure how to express that in gorm.